scripts/download_and_create_docker_db_volume.sh

```diff
@@ -7,8 +7,8 @@
 
 VOLUME='mysql-data-volume'
 
-echo 'Downloading data'
-curl -O https://swefreq.nbis.se/static/testing/mysql-data.tar.bz2
+#echo 'Downloading data'
```

Comment:
And uncomment these lines (since it should only be run in case the data volume wasn't created).